composer.json

```diff
@@ -94,5 +94,15 @@
     "drupal/core-dev": "*",
     "drupal/coder": "^8.3",
     "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0"
+  },
+  "config": {
+    "allow-plugins": {
+      "cweagans/composer-patches": true,
+      "composer/installers": true,
+      "drupal/core-composer-scaffold": true,
+      "drupal/core-project-message": true,
+      "drupal/console-extend-plugin": true,
+      "dealerdirect/phpcodesniffer-composer-installer": true
+    }
```

Comment:
(This could be avoided by doing `composer install -y` but I think it's a good idea to just mark these as safe as part of the regular `composer.json` file)